service/kas/access/accessPdp.go

```diff
@@ -16,16 +16,18 @@ const (
 	ErrDecisionCountUnexpected = Error("authorization decision count unexpected")
 )
 
-func canAccess(ctx context.Context, entity *authorization.Entity, policy Policy, sdk *otdf.SDK) (bool, error) {
+func canAccess(ctx context.Context, token *authorization.Token, policy Policy, sdk *otdf.SDK) (bool, error) {
 	if len(policy.Body.Dissem) > 0 {
-		dissemAccess, err := checkDissems(policy.Body.Dissem, entity)
-		if err != nil {
-			return false, err
-		}
-		return dissemAccess, nil
+		// dissemAccess, err := checkDissems(policy.Body.Dissem, token)
+		// if err != nil {
+		// 	return false, err
+		// }
+
+		// throw an error, dont support dissems?
+		return false, errors.New("don't support dissems with v2 platform kas")
```

Comment:
can we remove dissems checks since were just using the token?

Reply:
@biscoe916 Dissems is deprecated in the spec last time I looked. Can we move forward with removing it? 

If we still need to support Dissems this probably isn't the right place for the check anymore. 

cc: @jrschumacher @ttschampel

Reply:
we should not drop dissems as eventually we will need it, or some equivalent - however, it could be moved somewhere else

Reply:
@elizabethhealy We discussed this today and you can remove the dissems check from kas. 

I created a place holder issue to track re-implementing it with the GetDecision call on the future.

https://github.com/opentdf/platform/issues/812